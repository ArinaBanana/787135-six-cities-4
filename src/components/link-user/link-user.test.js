import React from "react";
import renderer from "react-test-renderer";
import LinkUser from "./link-user.jsx";
import {MemoryRouter} from "react-router-dom";

it(`Should render link user`, () => {
  const tree = renderer.create(
      <MemoryRouter>
        <LinkUser email={`email`} />
      </MemoryRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
