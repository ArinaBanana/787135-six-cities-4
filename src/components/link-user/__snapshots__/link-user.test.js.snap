// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render link user 1`] = `
<a
  className="header__nav-link header__nav-link--profile"
  href="#"
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__user-name user__name"
  >
    email
  </span>
</a>
`;
